worker/utils/utils.js

```diff
@@ -180,6 +180,28 @@ module.exports = {
     }
     return returnObject;
   },
+  
+  async getSnootyProjectName(repoDirName){
```

Comment:
We can't actually just read snooty.toml for this, right? `get-project-name` just reads the makefile variable which has to match the snooty project name, right?

Reply:
Yeah we can't read snooty.toml because at the point in the autobuilder process where we need to have the project name, we haven't yet downloaded the repo (which contains the snooty.toml file)